### PR TITLE
Create anibalsolon.md

### PR DESCRIPTION
**Pull request by anibalsolon**

### CODE REVIEW

File: content/team/anibalsolon.md [hunk 1]
```diff
@@ -0,0 +1,10 @@
+---
+title: "Anibal Solon Heinsfeld"
+date: 2022-01-26T17:32:40Z
+draft: false
+position : "Graduate Research Assistant"
+description : "Anibal is a graduate research assistant in the brainlife.io project. His focus is Machine Learning methods applied in Neuroscience, developing methods that account for reproducibility and reliability. Besides his academic career, Anibal contributed to several open-source projects and loves computer infrastructure and programming. He joined the University of Texas at Austin in 2019 and joined the brainlife.io team in 2021."
```
Comment (francopestilli) [inline content/team/anibalsolon.md]: How about the following?
"Anibal is a graduate student in the Pestilli Lab. His focus is Machine Learning methods applied in Neuroscience, developing methods that account for reproducibility and reliability. Anibal has contributed to several open-source projects (ADD A FEW HERE) and loves computer infrastructure and programming. He joined the University of Texas at Austin in 2019 and joined the brainlife.io team in 2021."

** See ADD A FEW HERE

**Review by francopestilli — CHANGES_REQUESTED** [review]

File: content/team/anibalsolon.md [hunk 2]
```diff
@@ -0,0 +1,10 @@
...
+image : "/images/avatar/anibalsolon.jpg"
+tags : []
```
Comment (bhatiadheeraj) [inline content/team/anibalsolon.md]: tags: ['active']

This will add you to the active team :)

File: content/team/anibalsolon.md (hunk 1 above)
Comment (anibalsolon) [reply content/team/anibalsolon.md]: Thanks! I've added some.

File: content/team/anibalsolon.md (hunk 2 above)
Comment (anibalsolon) [reply content/team/anibalsolon.md]: Thanks! I've included the tag.

**Pull request merged**